dropwizard-client/src/main/java/io/dropwizard/client/JerseyClientConfiguration.java

```diff
@@ -12,7 +12,7 @@
  * {@link HttpClientConfiguration}.
  *
  * @see HttpClientConfiguration
- * @see <a href="http://dropwizard.io/0.9.1/docs/manual/configuration.html#jerseyclient">Jersey Client Configuration</a>
+ * @see <a href="http://dropwizard.io/1.0.1/docs/manual/configuration.html#jerseyclient">Jersey Client Configuration</a>
```

Comment:
Just updated it to 1.0.2 and I agree there should be a non version specific link